cameronRomo/my-strain-book
# Feature/configure app

### What’s this PR do?

* Configures react app and adds fundamental components

### Where should the reviewer start?

* `src/App/App.js` line 1
*  `src/Card/Card.js` line 1
* `src/Experiences/Experiences.js`  line 1 (set up for next branch)
* `src/NavBar/NavBar.js` line 1
*  `src/Strains/Strains.js` line 1


### How should this be manually tested?

* Pull down and run `npm start`, then select a strain and see pertinent data render on the page

### Any background context you want to provide?

* React with hooks is the method for keeping state

### Screenshots (if appropriate):

![Screen Shot 2021-01-15 at 7 43 21 PM](https://user-images.githubusercontent.com/63012953/104795246-f47dcd80-5769-11eb-9b80-64b99cf25055.png)
